### PR TITLE
Use QtConcurrent in DynamicTexture, remove opemmp dependency

### PR DESCRIPTION
**Pull request by rdumusc**
The resulting code processes 300MB and 1.5GB test images ~20% faster on average

### CODE REVIEW

File: tide/core/DynamicTexture.cpp [hunk 1]
```diff
@@ -50,49 +50,48 @@
...
+DynamicTexture::DynamicTexture( DynamicTexturePtr parent_,
+                                const int childIndex )
+    : _imageExtension( parent_->_imageExtension )
+    , _useImagePyramid( false )
+    , _parent( parent_ )
+    , _imageCoordsInParentImage( imageBounds[childIndex] )
+    , _depth( parent_->_depth + 1 )
+{
+    // append childIndex to parent's path to form this object's path
+    _treePath = parent_->_treePath;
```
Comment (bilgili) [inline tide/core/DynamicTexture.cpp]: _treePath( parent_->_treePAth ) to the initialization list ?

File: tide/core/DynamicTexture.cpp [hunk 2]
```diff
@@ -235,19 +234,14 @@ bool DynamicTexture::writePyramidImagesRecursive( const QString& pyramidFolder )
...
-        imageBounds[2] = QRectF( 0.5, 0.5, 0.5, 0.5 );
-        imageBounds[3] = QRectF( 0.0, 0.5, 0.5, 0.5 );
-
-#pragma omp parallel for
+        QList<DynamicTexturePtr> children;
         for( unsigned int i=0; i<4; ++i )
-        {
-            DynamicTexturePtr child( new DynamicTexture( "", shared_from_this(),
-                                                         imageBounds[i], i ));
+            children.push_back( DynamicTexturePtr(
```
Comment (bilgili) [inline tide/core/DynamicTexture.cpp]: emplace_back( new DynamicTexture( shared_from_this(), i ) ) if you are C++11. Just nicer and without copy construction.

File: tide/core/DynamicTexture.cpp (hunk 1 above)
Comment (rdumusc) [inline tide/core/DynamicTexture.cpp]: done

File: tide/core/DynamicTexture.cpp (hunk 2 above)
Comment (rdumusc) [inline tide/core/DynamicTexture.cpp]: good idea, but does not work with QList

**Pull request merged**